api/v1/models/testimonial.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python3
+"""
+Testimonial data model
+map testimonial table into 
+a python object (Testimonial class)
+"""
+from sqlalchemy import (
+    create_engine,
+    Column,
+    Integer,
+    String,
+    Text,
+    DateTime,
+    ForeignKey,
+    func
+)
+from sqlalchemy.orm import relationship
+from datetime import datetime
+from api.v1.models.base import Base
+from api.v1.models.base_model import BaseModel
+from sqlalchemy.dialects.postgresql import UUID
+
+class Testimonial(BaseModel, Base):
+    __tablename__ = 'testimonials'
+
+    firstname = Column(String(50), nullable=False)
+    lastname = Column(String(50), nullable=False)
+    content = Column(Text, nullable=False)
+    created_at = Column(DateTime(timezone=True), server_default=func.now())
+    updated_at = Column(DateTime(timezone=True), server_default=func.now())
```

Comment:
`created_at` and `updated_at` already exists in the `BaseModel` class, when you've adjusted this, make a new migration for the table and push

Reply:
Yes thats true